Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:8.4.0
+
+# Install packages
+RUN npm install -g yarn
+
+# Set up deploy user and working directory
```

Comment:
Normally we could safely run as `root` in a container (as processes in a container are "sandboxed" and have no system-level root privileges by default but I encountered an error whereby node-foreman tried to drop sudo and failed: https://github.com/strongloop/node-foreman/issues/70